src/signs/ui/display/SignDisplay.tsx

```diff
@@ -1,10 +1,121 @@
 import React from 'react'
-import { Col, Row } from 'react-bootstrap'
+import { Col, Container, Popover, Row } from 'react-bootstrap'
+import { RouteComponentProps } from 'react-router-dom'
+import withData, { WithoutData } from 'http/withData'
+import SignsService from 'signs/application/SignsService'
+import Sign from 'signs/domain/Sign'
+import HelpTrigger from 'common/HelpTrigger'
+import _ from 'lodash'
+import './signDisplay.css'
+import MesZlContent from 'signs/ui/search/MesZLContent'
 
-export default function SignDisplay(): JSX.Element {
+function CuneiformFontsHelpPopover(): JSX.Element {
   return (
-    <Row className="mt-5">
-      <Col className="text-center mt-5">Under Construction</Col>
+    <Popover
+      id={_.uniqueId('CuneiformFontsHelpHelp-')}
+      title="Cuneiform Form Help"
+    >
+      <Popover.Content>
+        <div>Cuneiform fonts by S. Vanserveren </div>
+      </Popover.Content>
+    </Popover>
+  )
+}
+
+function SignHeading({
+  signName,
+  cuneiformLetters,
+}: {
+  signName: string
+  cuneiformLetters: string
+}): JSX.Element {
+  const CuneiformFonts = ({
+    font = undefined,
```

Comment:
Use `null` instead of `undefined`.

The font is optional why set the default to a null value?

Reply:
i made the default to an empty string.